src/frontend/ocamlmerlin/ocamlmerlin_server.ml

```diff
@@ -64,6 +64,8 @@ module Server = struct
     | None ->
       Logger.log ~section:"server" ~title:"cannot setup listener" ""
     | Some server ->
+      (* If the client closes its connection, don't let it kill us with a SIGPIPE. *)
+      if Sys.unix then ignore @@ Sys.signal Sys.sigpipe Sys.Signal_ignore;
```

Comment:
I think that you can use `set_signal` here to remove the need for ignore.

Reply:
@liam923 gentle ping :-)

Reply:
Sorry about that, just fixed it

Reply:
Thanks !